docs/guides/frontend-redux.md

```diff
@@ -0,0 +1,63 @@
+# Frontend Data Architecture
+
+This post describes the architecture used throughout [frontend-scaffold](https://github.com/Code-4-Community/frontend-scaffold), Code4Community's shared frontend template used by all client projects. The scaffold is a React application created with Create React App using the TypeScript template. It uses Redux for application state management. 
+
+## What is Redux? 
+
+Redux is a predictable state container for JavaScript apps.
```

Comment:
What does this mean here (should 'predictable state container' be known already by frontend devs)? My only interaction with containers has been through docker

Reply:
^^

Reply:
It's different from Docker. This line I pulled directly from the Redux docs. The idea is that all of the application's state is contained within this library, as opposed to using multiple tools towards this goal. Some examples of that would be a React component's state, or storing things in local storage, or cookies... any way to store data in the app. 

It's "predictable" because Redux establishes rules for accessing data that are always followed - think of rules against mutation in OOD. If data is changed, it will never be unexpected.